src/convertToObject.js

```diff
@@ -7,6 +7,29 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+  const regex = /\n+|\t+/g;
```

Comment:
[CODE KNOWLEDGE] - The regex pattern you are using seems to replace newlines and tabs with spaces, which is unnecessary. Instead, you should focus on splitting the string into individual styles and trimming each one. The regex should be used to split the string by semicolons followed by optional whitespace and newlines.